### PR TITLE
fix: [workspace] File name filtering is missing

### PR DESCRIPTION
**Pull request by GongHeng2017**
The function of file name filtering is missing.

Log: solved problem of workspace
Bug: https://pms.uniontech.com/bug-view-192395.html 
         https://pms.uniontech.com/bug-view-192625.html

### CODE REVIEW

File: src/plugins/filemanager/core/dfmplugin-workspace/models/fileviewmodel.cpp [hunk 1]
```diff
@@ -726,3 +732,36 @@ void FileViewModel::changeState(ModelState newState)
     state = newState;
     Q_EMIT stateChanged();
 }
+
+bool FileViewModel::passNameFilters(const AbstractFileInfoPointer &info) const
+{
+    if (filterSortWorker->getNameFilters().isEmpty())
```
Comment (Lighto-Ku) [inline src/plugins/filemanager/core/dfmplugin-workspace/models/fileviewmodel.cpp]: 使用filterSortWorker前，最好先判空

Comment (GongHeng2017) [reply src/plugins/filemanager/core/dfmplugin-workspace/models/fileviewmodel.cpp]: ok

**Review by Lighto-Ku — COMMENTED** [review]
+1

**Review by max-lvs — APPROVED** [review]
+2

**Pull request merged**